### DISCUSSION TITLE
How to train a multi task network over the CAE rather than linear classifier

### DISCUSSION THREAD

**Discussion by letdivedeep** [open-mmlab/mmselfsup#672] [How-to]
How to train a multi task network over the CAE rather than linear classifier

**Comment by fangyixiao18** [comment]
did you mean train multi-task with CAE pretrained models?

**Reply by letdivedeep** [reply]
@fangyixiao18 yes ..

**Comment by mzr1996** [comment]
MMClassification has supported the multi-task training, but it's still an experimental functionality, and we will add documentation recently. If you are interested in it, please see the PR https://github.com/open-mmlab/mmclassification/pull/1229

**Reply by letdivedeep** [reply]
@mzr1996 bit confused on how to actually use this PR, 
For the multi-task network , based on my knowledge we may need to update the following things : 
1) Building a custom multi-task dataset and loader
2) Build the Custom Multi Task Network with multiple head 
3) Loss Function for the multiple branches 

can you guide me on how to get started with it if i want to create it with 2 heads
